chaoscenter/subscriber/subscriber.go

```diff
@@ -62,6 +64,9 @@ func init() {
 
 	var c Config
 
+	var subscriberGraphql = graphql.NewGqlServer()
+	var subscriberK8s = k8s.NewKubernetes(subscriberGraphql)
```

Comment:
If you are using local scoped variables with initialisation, the best practice is to define the variables with `:=` than`var xyz = abc()`

use `var` style definition only for global scoping, or when you want to lazy initialise a variable later on.
```suggestion
	subscriberGraphql := graphql.NewGqlServer()
	subscriberK8s := k8s.NewKubernetes(subscriberGraphql)
```